course/auth.py

```diff
@@ -333,15 +333,21 @@ def make_sign_in_key(user):
     return m.hexdigest()
 
 
-def check_sign_in_key(user_id, token):
-    users = get_user_model().objects.filter(
-            id=user_id, sign_in_key=token)
-
-    assert users.count() <= 1
-    if users.count() == 0:
-        return False
-
-    return True
+def logout_confirmation_required(
+        func, redirect_field_name=REDIRECT_FIELD_NAME,
+        logout_confirmation_url='relate-logout-confirmation'):
+    """
+    Decorator for views that checks that no user is logged in, or redirecting
+    to the logout_confirmation_url
```

Comment:
Rephrase: If a user is currently logged in, redirect them to the logout page.

Reply:
Thanks, updated.